Makefile

```diff
@@ -96,20 +96,73 @@ install-linux:
 install-darwin:
 	cp builds/lagoon-cli-${VERSION}-darwin-amd64 ${ARTIFACT_DESTINATION}/lagoon
 
-release-patch: 
+release-patch:
 	$(eval VERSION=$(shell ${PWD}/increment_ver.sh -p $(shell git describe --abbrev=0 --tags)))
 	git tag $(VERSION)
 	mkdocs gh-deploy
 	git push $(GIT_ORIGIN) main --tags
 
-release-minor: 
+release-minor:
 	$(eval VERSION=$(shell ${PWD}/increment_ver.sh -m $(shell git describe --abbrev=0 --tags)))
 	git tag $(VERSION)
 	mkdocs gh-deploy
 	git push $(GIT_ORIGIN) main --tags
 
-release-major: 
+release-major:
 	$(eval VERSION=$(shell ${PWD}/increment_ver.sh -M $(shell git describe --abbrev=0 --tags)))
 	git tag $(VERSION)
 	mkdocs gh-deploy
 	git push $(GIT_ORIGIN) main --tags
+
+api-tests: gen
+	GO111MODULE=on $(GOCMD) fmt ./...
+	GO111MODULE=on $(GOCMD) vet ./...
+	GO111MODULE=on $(GOCMD) test -v -run '(TestEnvironmentCommands|TestProjectCommands)' ./...
```

Comment:
You could update any of the tests that need to use the API to be called
```
TestAPIEnv...
TestAPIProject...
```
And then you can use this `-run '^(TestAPI)` instead.

And then for the existing `make test`, you can add `- run '^(?!TestAPI)(Test)'` and it will run every other test except the ones that need the API.

Reply:
Good call, I'll add this. I don't think the negative look aheads work with Go, but can tweak the regex.